unitpackage/electrochemistry/reference_electrodes.py

```diff
@@ -0,0 +1,248 @@
+r"""
+This module contains the :class: `ReferenceElectrode` which contains
+reference data for reference electrodes and allows shifting potentials
+between reference scales.
+
+EXAMPLES::
+
+    >>> from unitpackage.electrochemistry.reference_electrodes import ReferenceElectrodes
+    >>> ReferenceElectrodes["Ag/AgCl-sat"] # doctest: +NORMALIZE_WHITESPACE
+    ReferenceElectrode(name='Ag/AgCl-sat', fullName='KCl Saturated silver / silver chloride electrode',
+    entries=[ReferenceElectrodeEntry(value=0.197, unit='V', vs='SHE', type='experimental', temperature=None,
+    source={'isbn': '978-1119334064'}, uncertainty=None, preferred=True, choice=None, doi=None, isbn=None)],
+    alias=None, temperatureDependence=None)
+
+    >>> ReferenceElectrodes.convert(0.55, "Ag/AgCl-sat", "SHE")
+    0.35300000000000004
+
+"""
+
+# ********************************************************************
+#  This file is part of unitpackage.
+#
+#        Copyright (C) 2025 Johannes Hermann
+#        Copyright (C) 2025 Albert Engstfeld
+#
+#  unitpackage is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  unitpackage is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with unitpackage. If not, see <https://www.gnu.org/licenses/>.
+# ********************************************************************
+
+
+import logging
+from dataclasses import dataclass, field
+from typing import Any
+
+from unitpackage.electrochemistry.reference_electrode_data import (
+    REFERENCE_ELECTRODE_DATA,
+)
+
+logger = logging.getLogger("unitpackage")
+
+
+@dataclass(frozen=True)
+class ReferenceElectrodeEntry:  # pylint: disable=too-many-instance-attributes
+    """
+    Represents an entry with a value of a specified reference electrode.
```

Comment:
"entry" as in :class:`Entry`? Just trying to understand.

Reply:
No this is an entry within a reference electrode. So for each electrode one can find different values in the literature. Each entry provides such a value, including its source and additional details.